thrift/src/main/thrift/datanode.thrift

```diff
@@ -118,14 +118,16 @@ struct TSendPlanNodeResp {
   3: optional common.TSStatus status
 }
 
-struct TFetchFragmentInstanceStateReq {
+struct TFetchFragmentInstanceInfoReq {
   1: required TFragmentInstanceId fragmentInstanceId
 }
 
 // TODO: need to supply more fields according to implementation
-struct TFragmentInstanceStateResp {
+struct TFragmentInstanceInfoResp {
   1: required string state
-  2: optional list<string> failedMessages
+  2: optional i64 endTime
+  3: optional list<string> failedMessages
```

Comment:
In case we just need failureMessage string rather than the full context. In fact, we could get failureMessage from the failureInfo, but I think set failure message as cache is alright~